doc/source/whatsnew/v1.0.2.rst

```diff
@@ -51,4 +58,4 @@ Bug fixes
 Contributors
 ~~~~~~~~~~~~
 
-.. contributors:: v1.0.1..v1.0.2|HEAD
\ No newline at end of file
+.. contributors:: v1.0.1..v1.0.2|HEAD
```

Comment:
what changed here?

Reply:
I think the original file lacked a newline at the end of file so I may have added it to placate git. Can remove if we want to clear up the diff.

Reply:
no it’s fine